apps/docs-snippets2/src/cookbook/deposit-and-withdraw/withdraw.ts

```diff
@@ -0,0 +1,34 @@
+import { Provider, Wallet } from 'fuels';
+
+import { LOCAL_NETWORK_URL, WALLET_PVT_KEY } from '../../env';
+import { LiquidityPoolFactory } from '../../typegend';
+
+const provider = await Provider.create(LOCAL_NETWORK_URL);
+const wallet = Wallet.fromPrivateKey(WALLET_PVT_KEY, provider);
+
+const deploy = await LiquidityPoolFactory.deploy(wallet, {
+  configurableConstants: {
+    TOKEN: { bits: provider.getBaseAssetId() },
+  },
+});
+
+const { contract } = await deploy.waitForResult();
+
+const depositAmount = 100_000;
+const liquidityOwner = Wallet.generate({ provider });
+
+// #region deposit-and-withdraw-cookbook-3
```

Comment:
Should this come before the imports?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3367/commits/aefcdf3b96c79de8f7f509928e00e3d2b3dd9f7c